shared/fs/row/still.js

```diff
@@ -1,152 +1,66 @@
 // @flow
 import * as React from 'react'
 import * as Types from '../../constants/types/fs'
-import {globalColors, globalMargins, isMobile, glamorous} from '../../styles'
-import rowStyles from './styles'
-import {Badge, Box, Box2, ClickableBox, Icon, Meta, Text} from '../../common-adapters'
-import PathItemIcon from '../common/path-item-icon'
+import {globalColors, globalMargins, isMobile} from '../../styles'
+import {rowStyles, StillCommon, type StillCommonProps} from './styles'
```

Comment:
I don't think `StillCommon` and `StillCommonProps` should be in a file/directory called `styles`.

Reply:
There's a TODO in `./styles` to rename it in `common.js`, and is addressed in the 6th of this batch of PRs (mechanical changes).